static/scripts/topicEdit.js

```diff
@@ -326,7 +326,7 @@ class TopicBlockList extends React.Component {
      * Render the list items.
      */
 	render() {
-		const neXboardEnabled = ($contentBlocksContainer.data('nexboardenabled') === true);
+		const neXboardButtonEnabled = ($contentBlocksContainer.data('neXboardButtonEnabled') === false);
```

Comment:
Shouldn't it be `=== true`? To me it looks that If the `neXboardButtonEnabled` will be `true` (in a case we do want to show the button) we will **not** show the button - it would have to be `false` which is not valid.